### ISSUE TITLE
[BUG] - SA2 scenario is slightly overconservative

### ISSUE THREAD

**Issue by dodu94** [JADE-V-V/JADE#321]
**Describe the bug**
The irradiation scenario (SA2) used in the D1S assessment to compute irradiation factors is not entirely correct (conservative)

**Expected behavior**
Either this is corrected or the inconsistency needs to be mentioned in the documentation

**Comment by alexvalentine94** [comment]
Agreed - for the scope of what is trying to be achieved here I think this is acceptable. The important point is that the same scenario is being used for all the simulations.
